### ISSUE TITLE
ffmpeg: Fix version conflicts

### ISSUE THREAD

**Issue by madebr** [bincrafters/community#854]
openjpeg/2.3.0 has libpng/1.6.34
openjpeg/2.3.1 has libpng/1.6.37
freetype/2.10.0 has libpng/1.6.37
ffmpeg/4.1.0 has openjpeg/2.3.0
==> having as a dependency freetype and ffmpeg will let libpng/1.6.34 clash with libpng/1.6.37
FIX: update openjpeg dependency to 2.3.1: https://github.com/bincrafters/conan-ffmpeg/pull/5

qt/5.13.0 has OpenSSL/1.1.1c
ffmpeg/4.1.0 has OpenSSL/1.1.1b
==> having as a dependency qt and ffmpeg will let OpenSSL/1.1.1b clash with OpenSSL/1.1.1c
FIX: update OpenSSL dependency to 1.1.1c: https://github.com/bincrafters/conan-ffmpeg/pull/6

Fixing this error is a bit cumbersome, imho.
How do you handle these problems?

**Comment by Croydon** [comment]
Merged.

We can't automate stuff like this right now. So we need to handle it manually for the foreseeable future.

OpenSSL and other security relevant packages need to stay as up-to-date as possible, that we are getting them out-of-sync manually by over > 400 packages is not surprising.

**Comment by madebr** [comment]
Using a similar script as in #828,
I've parsed the dependencies of all packages of bincrafters.

e.g. the following should be read as:
- `OpenSSL/1.0.2o` is required by `conan-wt`, `conan-srt`, `conan-mosquitto`...
- `OpenSSL/1.0.2p` is required by `conan-folly`
- etc.
```
OpenSSL:
  1.0.2o:
  - conan-wt
  - conan-srt
  - conan-mosquitto
  - conan-mongo-c-driver
  - conan-botan
  1.0.2p:
  - conan-folly
  1.0.2q:
  - conan-libtins
  - conan-caf
  1.0.2r:
  - conan-thrift
  - conan-opusfile
  - conan-nghttp2
  - conan-libzip
  - conan-libevent
  - conan-cpprestsdk
  1.0.2s:
  - conan-mysql-connector-c
  - conan-libwebsockets
  - conan-libpq
  1.0.2x_latest:
  - conan-asio
  1.1.1b:
  - conan-libcurl
  1.1.1c:
  - conan-websocketpp
  - conan-qt
  - conan-ffmpeg
  latest_1.0.2x:
  - conan-libssh2
```

complete output (using the most recent stable branch of every repo):
```
7z_installer:
  '1.0':
  - conan-sigcpp
  - conan-ruby_installer
  - conan-msys2_installer
  - conan-cairo
Azure-C-Shared-Utility:
  1.0.46:
  - conan-azure-umqtt-c
  - conan-azure-uhttp-c
  - conan-azure-uamqp-c
Boost.Accumulators:
  1.65.1:
  - conan-boost
Boost.Algorithm:
  1.65.1:
  - conan-boost
Boost.Align:
  1.65.1:
  - conan-boost
Boost.Any:
  1.65.1:
  - conan-boost
Boost.Array:
  1.65.1:
  - conan-boost
Boost.Asio:
  1.65.1:
  - conan-boost
Boost.Assert:
  1.65.1:
  - conan-boost
Boost.Assign:
  1.65.1:
  - conan-boost
Boost.Atomic:
  1.65.1:
  - conan-boost
Boost.Bimap:
  1.65.1:
  - conan-boost
Boost.Bind:
  1.65.1:
  - conan-boost
Boost.Chrono:
  1.65.1:
  - conan-boost
Boost.Circular_Buffer:
  1.65.1:
  - conan-boost
Boost.Compatibility:
  1.65.1:
  - conan-boost
Boost.Compute:
  1.65.1:
  - conan-boost
Boost.Concept_Check:
  1.65.1:
  - conan-boost
Boost.Config:
  1.65.1:
  - conan-boost
Boost.Container:
  1.65.1:
  - conan-boost
Boost.Context:
  1.65.1:
  - conan-boost
Boost.Conversion:
  1.65.1:
  - conan-boost
Boost.Convert:
  1.65.1:
  - conan-boost
Boost.Core:
  1.65.1:
  - conan-boost
Boost.Coroutine:
  1.65.1:
  - conan-boost
Boost.Coroutine2:
  1.65.1:
  - conan-boost
Boost.Crc:
  1.65.1:
  - conan-boost
Boost.Date_Time:
  1.65.1:
  - conan-boost
Boost.Detail:
  1.65.1:
  - conan-boost
Boost.Disjoint_Sets:
  1.65.1:
  - conan-boost
Boost.Dll:
  1.65.1:
  - conan-boost
Boost.Dynamic_Bitset:
  1.65.1:
  - conan-boost
Boost.Endian:
  1.65.1:
  - conan-boost
Boost.Exception:
  1.65.1:
  - conan-boost
Boost.Fiber:
  1.65.1:
  - conan-boost
Boost.Filesystem:
  1.65.1:
  - conan-boost
Boost.Flyweight:
  1.65.1:
  - conan-boost
Boost.Foreach:
  1.65.1:
  - conan-boost
Boost.Format:
  1.65.1:
  - conan-boost
Boost.Function:
  1.65.1:
  - conan-boost
Boost.Function_Types:
  1.65.1:
  - conan-boost
Boost.Functional:
  1.65.1:
  - conan-boost
Boost.Fusion:
  1.65.1:
  - conan-boost
Boost.Generator:
  1.65.1:
  - conan-boost
Boost.Geometry:
  1.65.1:
  - conan-boost
Boost.Gil:
  1.65.1:
  - conan-boost
Boost.Graph:
  1.65.1:
  - conan-boost
Boost.Graph_Parallel:
  1.65.1:
  - conan-boost
Boost.Hana:
  1.65.1:
  - conan-boost
Boost.Heap:
  1.65.1:
  - conan-boost
Boost.Icl:
  1.65.1:
  - conan-boost
Boost.Integer:
  1.65.1:
  - conan-boost
Boost.Interprocess:
  1.65.1:
  - conan-boost
Boost.Intrusive:
  1.65.1:
  - conan-boost
Boost.Io:
  1.65.1:
  - conan-boost
Boost.Iostreams:
  1.65.1:
  - conan-boost
Boost.Iterator:
  1.65.1:
  - conan-boost
Boost.Lambda:
  1.65.1:
  - conan-boost
Boost.Level5Group:
  1.65.1:
  - conan-boost
Boost.Lexical_Cast:
  1.65.1:
  - conan-boost
Boost.Local_Function:
  1.65.1:
  - conan-boost
Boost.Locale:
  1.65.1:
  - conan-boost
Boost.Lockfree:
  1.65.1:
  - conan-boost
Boost.Log:
  1.65.1:
  - conan-boost
Boost.Logic:
  1.65.1:
  - conan-boost
Boost.Math:
  1.65.1:
  - conan-boost
Boost.Metaparse:
  1.65.1:
  - conan-boost
Boost.Move:
  1.65.1:
  - conan-boost
Boost.Mpi:
  1.65.1:
  - conan-boost
Boost.Mpl:
  1.65.1:
  - conan-boost
Boost.Msm:
  1.65.1:
  - conan-boost
Boost.Multi_Array:
  1.65.1:
  - conan-boost
Boost.Multi_Index:
  1.65.1:
  - conan-boost
Boost.Multiprecision:
  1.65.1:
  - conan-boost
Boost.Numeric_Conversion:
  1.65.1:
  - conan-boost
Boost.Numeric_Interval:
  1.65.1:
  - conan-boost
Boost.Numeric_Odeint:
  1.65.1:
  - conan-boost
Boost.Numeric_Ublas:
  1.65.1:
  - conan-boost
Boost.Optional:
  1.65.1:
  - conan-boost
Boost.Parameter:
  1.65.1:
  - conan-boost
Boost.Phoenix:
  1.65.1:
  - conan-boost
Boost.Poly_Collection:
  1.65.1:
  - conan-boost
Boost.Polygon:
  1.65.1:
  - conan-boost
Boost.Pool:
  1.65.1:
  - conan-boost
Boost.Predef:
  1.65.1:
  - conan-boost
Boost.Preprocessor:
  1.65.1:
  - conan-boost
Boost.Process:
  1.65.1:
  - conan-boost
Boost.Program_Options:
  1.65.1:
  - conan-boost
Boost.Property_Map:
  1.65.1:
  - conan-boost
Boost.Property_Tree:
  1.65.1:
  - conan-boost
Boost.Proto:
  1.65.1:
  - conan-boost
Boost.Ptr_Container:
  1.65.1:
  - conan-boost
Boost.Python:
  1.65.1:
  - conan-boost
Boost.Qvm:
  1.65.1:
  - conan-boost
Boost.Random:
  1.65.1:
  - conan-boost
Boost.Range:
  1.65.1:
  - conan-boost
Boost.Ratio:
  1.65.1:
  - conan-boost
Boost.Rational:
  1.65.1:
  - conan-boost
Boost.Regex:
  1.65.1:
  - conan-boost
Boost.Scope_Exit:
  1.65.1:
  - conan-boost
Boost.Serialization:
  1.65.1:
  - conan-boost
Boost.Signals:
  1.65.1:
  - conan-boost
Boost.Signals2:
  1.65.1:
  - conan-boost
Boost.Smart_Ptr:
  1.65.1:
  - conan-boost
Boost.Sort:
  1.65.1:
  - conan-boost
Boost.Spirit:
  1.65.1:
  - conan-boost
Boost.Stacktrace:
  1.65.1:
  - conan-boost
Boost.Statechart:
  1.65.1:
  - conan-boost
Boost.Static_Assert:
  1.65.1:
  - conan-boost
Boost.System:
  1.65.1:
  - conan-boost
Boost.Test:
  1.65.1:
  - conan-boost
Boost.Thread:
  1.65.1:
  - conan-boost
Boost.Throw_Exception:
  1.65.1:
  - conan-boost
Boost.Timer:
  1.65.1:
  - conan-boost
Boost.Tokenizer:
  1.65.1:
  - conan-boost
Boost.Tti:
  1.65.1:
  - conan-boost
Boost.Tuple:
  1.65.1:
  - conan-boost
Boost.Type_Erasure:
  1.65.1:
  - conan-boost
Boost.Type_Index:
  1.65.1:
  - conan-boost
Boost.Type_Traits:
  1.65.1:
  - conan-boost
Boost.Typeof:
  1.65.1:
  - conan-boost
Boost.Units:
  1.65.1:
  - conan-boost
Boost.Unordered:
  1.65.1:
  - conan-boost
Boost.Utility:
  1.65.1:
  - conan-boost
Boost.Uuid:
  1.65.1:
  - conan-boost
Boost.Variant:
  1.65.1:
  - conan-boost
Boost.Vmd:
  1.65.1:
  - conan-boost
Boost.Wave:
  1.65.1:
  - conan-boost
Boost.Winapi:
  1.65.1:
  - conan-boost
Boost.Xpressive:
  1.65.1:
  - conan-boost
Expat:
  2.2.6:
  - conan-wxwidgets
OpenSSL:
  1.0.2o:
  - conan-wt
  - conan-srt
  - conan-mosquitto
  - conan-mongo-c-driver
  - conan-botan
  1.0.2p:
  - conan-folly
  1.0.2q:
  - conan-libtins
  - conan-caf
  1.0.2r:
  - conan-thrift
  - conan-opusfile
  - conan-nghttp2
  - conan-libzip
  - conan-libevent
  - conan-cpprestsdk
  1.0.2s:
  - conan-mysql-connector-c
  - conan-libwebsockets
  - conan-libpq
  1.0.2x_latest:
  - conan-asio
  1.1.1b:
  - conan-libcurl
  1.1.1c:
  - conan-websocketpp
  - conan-qt
  - conan-ffmpeg
  latest_1.0.2x:
  - conan-libssh2
Parson:
  0.1.0:
  - conan-azure-iot-sdk-c
asio:
  1.12.0:
  - conan-websocketpp
  - conan-restinio
automake_build_aux:
  1.16.1:
  - conan-bison
bison:
  3.0.5:
  - conan-thrift
bison_installer:
  3.2.4:
  - conan-xkbcommon
  3.3.2:
  - conan-thrift
  - conan-libpcap
  - conan-gstreamer
boost:
  1.67.0:
  - conan-folly
  1.68.0:
  - conan-nghttp2
  1.69.0:
  - conan-websocketpp
  - conan-thrift
  - conan-msgpack
  - conan-asio
boost_algorithm:
  1.66.0:
  - conan-wt
boost_any:
  1.69.0:
  - conan-libtins
boost_array:
  1.66.0:
  - conan-wt
boost_asio:
  1.66.0:
  - conan-wt
  - conan-azmq
  1.69.0:
  - conan-nghttp2
  - conan-cpprestsdk
boost_assert:
  1.66.0:
  - conan-boost_level5group
boost_assign:
  1.66.0:
  - conan-wt
boost_atomic:
  1.69.0:
  - conan-cpprestsdk
boost_base:
  1.67.0:
  - conan-boost_level8group
  - conan-boost_level14group
  - conan-boost_level11group
  1.68.0:
  - conan-boost_signals
  1.69.0:
  - conan-boost_yap
  - conan-boost_xpressive
  - conan-boost_winapi
  - conan-boost_wave
  - conan-boost_vmd
  - conan-boost_variant
  - conan-boost_uuid
  - conan-boost_utility
  - conan-boost_unordered
  - conan-boost_units
  - conan-boost_type_traits
  - conan-boost_type_index
  - conan-boost_type_erasure
  - conan-boost_typeof
  - conan-boost_tuple
  - conan-boost_tti
  - conan-boost_tokenizer
  - conan-boost_timer
  - conan-boost_throw_exception
  - conan-boost_thread
  - conan-boost_test
  - conan-boost_system
  - conan-boost_static_assert
  - conan-boost_statechart
  - conan-boost_stacktrace
  - conan-boost_spirit
  - conan-boost_sort
  - conan-boost_smart_ptr
  - conan-boost_signals2
  - conan-boost_serialization
  - conan-boost_scope_exit
  - conan-boost_safe_numerics
  - conan-boost_regex
  - conan-boost_rational
  - conan-boost_ratio
  - conan-boost_range
  - conan-boost_random
  - conan-boost_qvm
  - conan-boost_python
  - conan-boost_ptr_container
  - conan-boost_proto
  - conan-boost_property_tree
  - conan-boost_property_map
  - conan-boost_program_options
  - conan-boost_process
  - conan-boost_preprocessor
  - conan-boost_predef
  - conan-boost_pool
  - conan-boost_poly_collection
  - conan-boost_polygon
  - conan-boost_phoenix
  - conan-boost_parameter_python
  - conan-boost_parameter
  - conan-boost_optional
  - conan-boost_numeric_ublas
  - conan-boost_numeric_odeint
  - conan-boost_numeric_interval
  - conan-boost_numeric_conversion
  - conan-boost_multi_index
  - conan-boost_multi_array
  - conan-boost_multiprecision
  - conan-boost_msm
  - conan-boost_mpl
  - conan-boost_mpi
  - conan-boost_mp11
  - conan-boost_move
  - conan-boost_metaparse
  - conan-boost_math
  - conan-boost_logic
  - conan-boost_log
  - conan-boost_lockfree
  - conan-boost_local_function
  - conan-boost_locale
  - conan-boost_lexical_cast
  - conan-boost_lambda
  - conan-boost_iterator
  - conan-boost_iostreams
  - conan-boost_io
  - conan-boost_intrusive
  - conan-boost_interprocess
  - conan-boost_integer
  - conan-boost_icl
  - conan-boost_hof
  - conan-boost_heap
  - conan-boost_hana
  - conan-boost_graph_parallel
  - conan-boost_graph
  - conan-boost_gil
  - conan-boost_geometry
  - conan-boost_fusion
  - conan-boost_function_types
  - conan-boost_functional
  - conan-boost_function
  - conan-boost_format
  - conan-boost_foreach
  - conan-boost_flyweight
  - conan-boost_filesystem
  - conan-boost_fiber
  - conan-boost_exception
  - conan-boost_endian
  - conan-boost_dynamic_bitset
  - conan-boost_dll
  - conan-boost_disjoint_sets
  - conan-boost_detail
  - conan-boost_date_time
  - conan-boost_cycle_group_d
  - conan-boost_cycle_group_c
  - conan-boost_cycle_group_b
  - conan-boost_cycle_group_a
  - conan-boost_crc
  - conan-boost_coroutine2
  - conan-boost_coroutine
  - conan-boost_core
  - conan-boost_convert
  - conan-boost_conversion
  - conan-boost_contract
  - conan-boost_context
  - conan-boost_container_hash
  - conan-boost_container
  - conan-boost_config
  - conan-boost_concept_check
  - conan-boost_compute
  - conan-boost_compatibility
  - conan-boost_circular_buffer
  - conan-boost_chrono
  - conan-boost_callable_traits
  - conan-boost_bind
  - conan-boost_bimap
  - conan-boost_beast
  - conan-boost_atomic
  - conan-boost_assign
  - conan-boost_assert
  - conan-boost_asio
  - conan-boost_array
  - conan-boost_any
  - conan-boost_align
  - conan-boost_algorithm
  - conan-boost_accumulators
boost_bind:
  1.66.0:
  - conan-wt
  - conan-boost_level5group
boost_build:
  1.69.0:
  - conan-boost_generator
boost_chrono:
  1.69.0:
  - conan-cpprestsdk
boost_config:
  1.66.0:
  - conan-wt
  - conan-boost_level5group
boost_core:
  1.66.0:
  - conan-boost_level5group
boost_date_time:
  1.69.0:
  - conan-cpprestsdk
boost_filesystem:
  1.66.0:
  - conan-wt
  1.69.0:
  - conan-cpprestsdk
boost_format:
  1.66.0:
  - conan-azmq
boost_fusion:
  1.66.0:
  - conan-wt
boost_generator:
  1.66.0:
  - conan-boost_level5group
  1.69.0:
  - conan-boost_base
boost_icl:
  1.69.0:
  - conan-libtins
boost_integer:
  1.66.0:
  - conan-wt
  - conan-boost_level5group
boost_interprocess:
  1.66.0:
  - conan-wt
boost_lexical_cast:
  1.66.0:
  - conan-wt
boost_logic:
  1.66.0:
  - conan-azmq
boost_math:
  1.66.0:
  - conan-wt
boost_move:
  1.66.0:
  - conan-boost_level5group
boost_multi_index:
  1.66.0:
  - conan-wt
boost_numeric_ublas:
  1.66.0:
  - conan-wt
boost_optional:
  1.66.0:
  - conan-wt
  1.67.0:
  - conan-mongo-cxx-driver
boost_package_tools:
  1.66.0:
  - conan-boost_level5group
boost_phoenix:
  1.66.0:
  - conan-wt
boost_pool:
  1.66.0:
  - conan-wt
  - conan-wt
boost_predef:
  1.66.0:
  - conan-boost_level5group
boost_preprocessor:
  1.66.0:
  - conan-boost_level5group
boost_program_options:
  1.66.0:
  - conan-wt
boost_python:
  1.67.0:
  - conan-boost_level14group
  '{ver}':
  - conan-boost_cycle_group_d
boost_random:
  1.66.0:
  - conan-azmq
  1.69.0:
  - conan-cpprestsdk
boost_range:
  1.66.0:
  - conan-wt
boost_regex:
  1.66.0:
  - conan-azmq
  1.69.0:
  - conan-cpprestsdk
boost_serialization:
  1.66.0:
  - conan-wt
boost_smart_ptr:
  1.66.0:
  - conan-wt
  - conan-boost_level5group
  1.67.0:
  - conan-mongo-cxx-driver
boost_spirit:
  1.66.0:
  - conan-wt
boost_static_assert:
  1.66.0:
  - conan-boost_level5group
boost_system:
  1.66.0:
  - conan-wt
  - conan-azmq
  1.67.0:
  - conan-mongo-cxx-driver
  1.69.0:
  - conan-nghttp2
  - conan-cpprestsdk
boost_thread:
  1.66.0:
  - conan-wt
  - conan-azmq
  1.69.0:
  - conan-nghttp2
  - conan-cpprestsdk
boost_throw_exception:
  1.66.0:
  - conan-boost_level5group
boost_tokenizer:
  1.66.0:
  - conan-wt
boost_tuple:
  1.66.0:
  - conan-wt
boost_type_traits:
  1.66.0:
  - conan-boost_level5group
boost_utility:
  1.67.0:
  - conan-mongo-cxx-driver
boost_variant:
  1.66.0:
  - conan-wt
bzip2:
  1.0.6:
  - conan-pcre2
  - conan-pcre
  - conan-libzip
  - conan-imagemagick
  - conan-freetype
  - conan-ffmpeg
  - conan-botan
  - conan-boost_cycle_group_c
c-ares:
  1.14.0:
  - conan-mosquitto
  1.15.0:
  - conan-nghttp2
cctz:
  '2.2':
  - conan-abseil
cmake_findboost_modular:
  1.66.0:
  - conan-wt
  - conan-azmq
  1.69.0:
  - conan-cpprestsdk
cpython:
  3.7.2:
  - conan-bullet3
cygwin_installer:
  2.9.0:
  - conan-libx264
  - conan-libiconv
  - conan-giflib
date:
  2.4.1:
  - conan-sqlpp11
dirent-win32:
  1.23.2:
  - conan-fswatch
double-conversion:
  3.1.1:
  - conan-folly
  3.1.4:
  - conan-qt
flac:
  1.3.2:
  - conan-sfml
  - conan-sdl2_mixer
flex:
  2.6.4:
  - conan-thrift
flex_installer:
  2.6.4:
  - conan-thrift
  - conan-libselinux
  - conan-libpcap
  - conan-gstreamer
fluidsynth:
  2.0.5:
  - conan-sdl2_mixer
fmt:
  5.2.0:
  - conan-restinio
  5.3.0:
  - conan-spdlog
freetype:
  2.10.0:
  - conan-qt
  - conan-ffmpeg
  2.9.0:
  - conan-sfml
  2.9.1:
  - conan-sdl2_ttf
  - conan-imagemagick
  - conan-harfbuzz
  - conan-cairo
gflags:
  2.2.1:
  - conan-glog
  - conan-folly
giflib:
  5.1.4:
  - conan-leptonica
glib:
  2.58.3:
  - conan-qt
  - conan-gstreamer
  - conan-fluidsynth
glog:
  '20181109':
  - conan-folly
gmp:
  6.1.2:
  - conan-nettle
  - conan-mpfr
gyp_installer:
  '20190423':
  - conan-libuv
harfbuzz:
  2.4.0:
  - conan-qt
http-parser:
  2.8.1:
  - conan-restinio
icu:
  '59.1':
  - conan-boost_locale
  '63.1':
  - conan-qt
  - conan-libxml2
  - conan-harfbuzz
  - conan-boost_regex
icu_installer:
  '%s':
  - conan-icu
intel_media_sdk:
  2018R2:
  - conan-ffmpeg
jansson:
  '2.12':
  - conan-nghttp2
jasper:
  2.0.14:
  - conan-opencv
jom_installer:
  1.1.2:
  - conan-qt
jsoncpp:
  1.0.0:
  - conan-jsonrpc-cpp
  1.8.4:
  - conan-libjson-rpc-cpp
khronos-opencl-headers:
  '20190412':
  - conan-khronos-opencl-icd-loader
  - conan-khronos-opencl-clhpp
khronos-opencl-icd-loader:
  '20190412':
  - conan-khronos-opencl-clhpp
lcms:
  '2.9':
  - conan-openjpeg
  - conan-imagemagick
leptonica:
  1.76.0:
  - conan-tesseract
libalsa:
  1.1.5:
  - conan-qt
  - conan-openal
  - conan-fluidsynth
libcurl:
  7.50.3:
  - conan-jsonrpc-cpp
  7.56.1:
  - conan-libjson-rpc-cpp
  - conan-date
libelf:
  0.8.13:
  - conan-libdwarf
  - conan-glib
libev:
  '4.25':
  - conan-nghttp2
libevent:
  2.1.8:
  - conan-thrift
  - conan-libwebsockets
  - conan-folly
libfdk_aac:
  0.1.5:
  - conan-ffmpeg
libffi:
  3.2.1:
  - conan-glib
libiconv:
  '1.15':
  - conan-sdl2
  - conan-qt
  - conan-log4cplus
  - conan-libxml2
  - conan-libqrencode
  - conan-ffmpeg
libjpeg:
  9c:
  - conan-wxwidgets
  - conan-sdl2_image
  - conan-qt
  - conan-leptonica
  - conan-imagemagick
libjpeg-turbo:
  1.5.2:
  - conan-wxwidgets
  - conan-sdl2_image
  - conan-opencv
libmad:
  0.15.1b:
  - conan-sdl2_mixer
libmicrohttpd:
  0.9.59:
  - conan-libjson-rpc-cpp
libmikmod:
  3.3.11.1:
  - conan-sdl2_mixer
libmodplug:
  0.8.9.0:
  - conan-sdl2_mixer
libmount:
  2.33.1:
  - conan-glib
libmp3lame:
  '3.100':
  - conan-ffmpeg
libmpg123:
  1.25.10:
  - conan-sdl2_mixer
libpcap:
  1.8.1:
  - conan-pcapplusplus
  - conan-libtins
libpng:
  1.6.34:
  - conan-opencv
  - conan-libqrencode
  - conan-leptonica
  1.6.36:
  - conan-sdl2_image
  - conan-cairo
  1.6.37:
  - conan-wxwidgets
  - conan-qt
  - conan-openjpeg
  - conan-imagemagick
  - conan-freetype
libpq:
  9.6.9:
  - conan-qt
  - conan-libpqxx
libselinux:
  '2.8':
  - conan-glib
libsodium:
  1.0.16:
  - conan-zmq
libssh2:
  1.8.0:
  - conan-libcurl
libtiff:
  4.0.9:
  - conan-wxwidgets
  - conan-sdl2_image
  - conan-openjpeg
  - conan-opencv
  - conan-leptonica
  - conan-imagemagick
libtirpc:
  1.1.4:
  - conan-libnsl
libusb:
  1.0.21:
  - conan-libpcap
libuuid:
  1.0.3:
  - conan-czmq
libuv:
  1.15.0:
  - conan-libwebsockets
libvpx:
  1.7.0:
  - conan-ffmpeg
libwebp:
  1.0.0:
  - conan-sdl2_image
  - conan-opencv
  - conan-leptonica
  - conan-imagemagick
  - conan-ffmpeg
libx264:
  '20171211':
  - conan-ffmpeg
libx265:
  '2.7':
  - conan-ffmpeg
libxml2:
  2.9.9:
  - conan-nghttp2
  - conan-imagemagick
lz4:
  1.8.3:
  - conan-folly
  - conan-czmq
lzma:
  5.2.3:
  - conan-ffmpeg
  5.2.4:
  - conan-libxml2
  - conan-libunwind
  - conan-imagemagick
  - conan-boost_cycle_group_c
m4_installer:
  1.4.18:
  - conan-gmp
  - conan-flex
  - conan-bison_installer
  - conan-bison
meson_installer:
  0.49.0:
  - conan-xkbcommon
  - conan-glib
  0.50.0:
  - conan-gstreamer
mongo-c-driver:
  1.11.0:
  - conan-mongo-cxx-driver
mozjpeg:
  3.3.1:
  - conan-wxwidgets
mpfr:
  4.0.2:
  - conan-mpc
msys2_installer:
  '20161025':
  - conan-pixman
  - conan-cairo
  latest:
  - conan-tcl
  - conan-pkg-config_installer
  - conan-mpdecimal
  - conan-m4_installer
  - conan-lzma
  - conan-lz4
  - conan-libmpg123
  - conan-libiconv
  - conan-libffi
  - conan-giflib
  - conan-flex
  - conan-ffmpeg
  - conan-bison_installer
  - conan-bison
mysql-connector-c:
  6.1.11:
  - conan-qt
nanomsg:
  1.1.2:
  - conan-cppnanomsg
nasm:
  2.13.01:
  - conan-libjpeg-turbo
  - conan-flac
nasm_installer:
  2.13.02:
  - conan-openh264
  - conan-mozjpeg
  - conan-libx265
  - conan-libx264
nghttp2:
  1.38.0:
  - conan-libcurl
ninja_installer:
  1.8.2:
  - conan-meson_installer
  - conan-libx265
  - conan-libuv
  - conan-czmq
nodejs_installer:
  10.15.0:
  - conan-emsdk_installer
npcap-dll:
  '0.94':
  - conan-npcap-wpcap
odbc:
  2.3.7:
  - conan-qt
ogg:
  1.3.3:
  - conan-vorbis
  - conan-theora
  - conan-sfml
  - conan-sdl2_mixer
  - conan-opusfile
  - conan-flac
openal:
  1.19.0:
  - conan-sfml
  - conan-qt
openexr:
  2.3.0:
  - conan-imagemagick
openh264:
  1.7.0:
  - conan-ffmpeg
openjpeg:
  2.3.0:
  - conan-leptonica
  2.3.1:
  - conan-imagemagick
  - conan-ffmpeg
openmpi:
  3.0.0:
  - conan-boost_level14group
  - conan-boost_cycle_group_d
opus:
  1.2.1:
  - conan-sdl2_mixer
  - conan-opusfile
  - conan-ffmpeg
opusfile:
  '0.10':
  - conan-sdl2_mixer
pcre:
  '8.41':
  - conan-glib
pcre2:
  '10.32':
  - conan-qt
  - conan-libselinux
  - conan-flex
pixman:
  0.38.0:
  - conan-cairo
pkg-config_installer:
  0.29.2:
  - conan-gstreamer
  - conan-glib
  - conan-fluidsynth
  - conan-cairo
portaudio:
  v190600.20161030:
  - conan-fluidsynth
premake_installer:
  4.4-beta5:
  - conan-soil2
protobuf:
  '{}':
  - conan-protoc_installer
pthread-win32:
  2.9.1:
  - conan-srt
  - conan-pcapplusplus
python_dev_config:
  '0.6':
  - conan-boost_python
readline:
  '7.0':
  - conan-fluidsynth
sdl2:
  2.0.7:
  - conan-ffmpeg
  2.0.9:
  - conan-smpeg
  - conan-sdl2_ttf
  - conan-sdl2_mixer
  - conan-sdl2_image
  - conan-qt
  - conan-fluidsynth
sqlite3:
  3.21.0:
  - conan-wt
  - conan-sqlite_orm
  3.25.3:
  - conan-sqlpp11-connector-sqlite3
  - conan-botan
  3.27.2:
  - conan-sqlitecpp
  3.28.0:
  - conan-qt
sqlpp11:
  '0.57':
  - conan-sqlpp11-connector-sqlite3
stb:
  '20180214':
  - conan-sfml
termcap:
  1.3.1:
  - conan-readline
tinymidi:
  '20130325':
  - conan-sdl2_mixer
vorbis:
  1.3.6:
  - conan-theora
  - conan-sfml
  - conan-sdl2_mixer
  - conan-ffmpeg
websocketpp:
  0.7.0:
  - conan-cpprestsdk
winflexbison:
  2.5.18:
  - conan-thrift
  - conan-thrift
winpcap:
  4.1.3:
  - conan-pcapplusplus
  - conan-libtins
xkbcommon:
  0.8.3:
  - conan-qt
yasm_installer:
  1.3.0:
  - conan-libmpg123
  - conan-ffmpeg
zlib:
  1.2.11:
  - conan-wxwidgets
  - conan-wt
  - conan-websocketpp
  - conan-thrift
  - conan-tcl
  - conan-sdl2_image
  - conan-ruby_installer
  - conan-qt
  - conan-protobuf
  - conan-pcre2
  - conan-pcre
  - conan-openjpeg
  - conan-opencv
  - conan-nghttp2
  - conan-mysql-connector-c
  - conan-mongo-c-driver
  - conan-libzip
  - conan-libxml2
  - conan-libwebsockets
  - conan-libtiff
  - conan-libssh2
  - conan-libpq
  - conan-libpng
  - conan-libdwarf
  - conan-libcurl
  - conan-leptonica
  - conan-imagemagick
  - conan-glib
  - conan-freetype
  - conan-folly
  - conan-ffmpeg
  - conan-cpprestsdk
  - conan-cairo
  - conan-botan
  - conan-boost_cycle_group_c
zmq:
  4.2.2:
  - conan-zmqpp
  - conan-ffmpeg
  - conan-cpperomq
  - conan-azmq
  4.2.5:
  - conan-czmq
  - conan-cppzmq
zstd:
  1.3.5:
  - conan-folly
  - conan-boost_cycle_group_c
'{dep}':
  '{ver}':
  - conan-boost_base
  - conan-boost_base
```

**Comment by Croydon** [comment]
Patch updates (the letters) should always be unproblementic. 

So, for example, if you know which packages are using 1.0.2o/p/q/r you could execute `bincrafters-conventions` on them and create pull requests. So we get them all to 1.0.2s.

If upgrades are possible, e.g. 1.0.2s -> 1.1.1c need to get checked manually. If upgrades are possible we are likely going to accept them as well.

Downgrades of any kinds to solve version conflicts should (probably) never be done.

**Comment by madebr** [comment]
Indeed, but imho the most recent testing branches of packages should have the most recent versions of its dependencies. Right now, the versions are a bit mixed.

**Comment by SSE4** [comment]
usually, I try to update OpenSSL versions, but unfortunately, they have API changes between 1.0.2 and 1.1.0, as well as between 1.1.0 and 1.1.1, so sometimes certain projects couldn't be updated easily, because they aren't adapted to API changes. therefore, we will always run into usage of multiple OpenSSL versions.

**Comment by madebr** [comment]
I understand the big difference and API change between 1.0x and 1.1x.
But right now there are 5 different 1.0x versions in use.
We should at least strive to have these versions merged to 1.0.2s (or 1.0.2x_latest) in their testing branch.

**Comment by Croydon** [comment]
> We should at least strive to have these versions merged to 1.0.2s (or 1.0.2x_latest) in their testing branch.

Nobody is arguing against that. But the few pull requests alone you just created have filled our Travis backlog with ~336 jobs. This might already fill the next 24h or so.

We just don't have the resources for high automation right now.

So again: If you run into version conflicts either try to override them in your project or create pull requests when upgrades are possible

**Comment by madebr** [comment]
Fair point.
My latest pull request are for adding support for the latest gcc version.
